nevergrad/functions/functionlib.py

```diff
@@ -62,6 +64,53 @@ def process(self, data: ArrayLike, deterministic: bool = True) -> np.ndarray:  #
     def _short_repr(self) -> str:
         return "Photonics"
 
+    
+# Example of ML problem.
+def _decision_tree_parametrization(depth: int):
+    # 10-folds cross-validation
+    num_data: int = 80
+    result: float = 0.
+    for cv in range(10):
+        list_X = []
+        list_X_test = []
+        for i in range(num_data):
+            if i % 10 != cv:
+                list_X += [[float(i)]]
```

Comment:
I'm pretty sure this could get created in a couple of lines at most,
but I don't get exactly what they are supposed to be, maybe a comment would be useful and I can find how to do this better

Reply:
maybe:
```python
X_all = np.arange(num_data)
X = X_all[(X_all % cv) > 0]
X_test = X_all[(X_all % cv) == 0]
```